botx/bot/bot.py

```diff
@@ -1046,6 +1050,26 @@ async def create_sticker_pack(self, *, bot_id: UUID, name: str) -> StickerPack:
 
         return botx_api_sticker_pack.to_domain()
 
+    async def delete_sticker_pack(self, *, bot_id: UUID, sticker_pack_id: UUID) -> None:
+        """Delete existing sticker pack.
+
+        Args:
+            bot_id: Bot which should perform the request.
+            sticker_pack_id: Target sticker pack.
+        """
+
+        method = DeleteStickerPackMethod(
+            bot_id,
+            self._httpx_client,
+            self._bot_accounts_storage,
+        )
+
+        payload = BotXAPIDeleteStickerPackRequestPayload.from_domain(
+            sticker_pack_id=sticker_pack_id,
```

Comment:
In `botx.bot.bot:Bot` all methods use positional arguments (except `delete_sticker` and `send_smartapp_notification`). In a separate commit unify all Bot methods to kwargs

Reply:
fixed